cms/envs/common.py

```diff
@@ -465,6 +467,8 @@
 USE_I18N = True
 USE_L10N = True
 
+STATICI18N_ROOT = PROJECT_ROOT / "static"
```

Comment:
This setting only defines where the JS files are generated. It's a directory that already exists for both CMS and LMS. We've looked at it on the sandbox and verified that the generated files are bundled up and hashed along with other static assets.